src/api/public/apidocs-new/paths/distributions_distribution_id.yaml

```diff
@@ -0,0 +1,58 @@
+get:
+  summary: Show a distribution.
+  description: |
+    Show a distribution that can be build against.
+  security:
```

Comment:
@jcronenberg you are missing the parameter part here, you have to add the `distribution_id`

Reply:
here an example https://github.com/openSUSE/open-build-service/blob/3853a4ce3d7311846529308bbed1d587d4ba9781/src/api/public/apidocs-new/paths/person_login_token.yaml#L6